frontend/css/brainzplayer.less

```
@@ -351,6 +351,23 @@
   }
 }
 
+.volume {
+  position: fixed;
+  width: fit-content;
+  z-index: 109;
+  height: 10em;
+  background-color: #f8f8f8;
+  right: 16em;
+  .volume-input {
```

Let's animate the transition between hidden and shown:
```suggestion
  bottom: -100px;
  transition: bottom 0.5s ease-in-out;
  .volume-input {
```

---

I also thought of including transition for 0.5s initially but then dropped it later 😅